pkg/handler/handler_test.go

```diff
@@ -84,6 +85,13 @@ func TestRegisterOpenAPIVersionedService(t *testing.T) {
 		if resp.StatusCode != tc.respStatus {
 			t.Errorf("Accept: %v: Unexpected response status code, want: %v, got: %v", tc.acceptHeader, tc.respStatus, resp.StatusCode)
 		}
+		if resp.StatusCode != 200 {
```

Comment:
I'll just mention that while I'm not super excited about this, I think it's fine given the current state of the test.